src/cf-tables/usage.md

```diff
@@ -1,14 +1,29 @@
 The cf-tables component formats tables, and provides an easy way to make tables sortable.
+The [`cf-core`](../core) component is a dependency of this component."
 
+<<<<<<< b6939a2f8eb00274fc8c71d8c555ae808ec48a9f
```

Comment:
Sorry, I think I saw all the lines for the same conflict and thought there were more